api/envoy/config/filter/network/ext_authz/v2/ext_authz.proto

```diff
@@ -7,22 +7,24 @@ import "envoy/api/v2/core/grpc_service.proto";
 
 import "validate/validate.proto";
 
-// [#not-implemented-hide:]
+// [#protodoc-title: Network External Authorization ]
+// The network layer external authorization service configuration :ref:`configuration overview <config_network_filters_ext_authz>`.
```

Comment:
nit: please line wrap to approximately 100 col (same elsewhere).

cc @htuch I think we used to do auto formatting/checking on this via clang-format on protos, but it might have been lost in the repo merge.